### PR TITLE
Supplement Documentation Checklist with Migration items and link to s…

### PR DESCRIPTION
**Pull request by jswope00**
…ubmit a github issue

Added a section for Migration-specific checklist items, rather than creating a new doc and duplicating items with the existing checklist.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jswope00!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: source/documentors/concepts/content_types.rst [hunk 1]
```diff
@@ -3,13 +3,30 @@ Documentation Content Types
 
 To keep documentation streamlined and make it easy for users to find answers, we make each topic one of four types:
 
-* **Quickstart** topics may seem similar to How-Tos but have a different focus.  Tutorials are specifically built for beginners and are meant to help them gain experience with the product.  For example taking a course through the entire course lifecycle, even if there is no content, and it's not exactly how you would do it with a real course, gives a beginner a meaningful experience that helps them better navigate the product.
+**Quickstart** 
 
-* **How-to** topics provide direct and easy-to-follow instructions to accomplish specific goals. For example, topics provide the steps to create a new course subsection and to schedule a course.
+Quickstart topics may seem similar to How-Tos but have a different focus.  Tutorials are specifically built for beginners and are meant to help them gain experience with the product.  For example taking a course through the entire course lifecycle, even if there is no content, and it's not exactly how you would do it with a real course, gives a beginner a meaningful experience that helps them better navigate the product.
```
[the remark this answers is not in the document]
Comment (sarina) [reply source/documentors/concepts/content_types.rst:8]: Should guidance be provided about document names, as the other 3 types below do?

File: source/documentors/concepts/content_types.rst [hunk 2]
```diff
@@ -3,13 +3,30 @@ Documentation Content Types
 
 To keep documentation streamlined and make it easy for users to find answers, we make each topic one of four types:
 
-* **Quickstart** topics may seem similar to How-Tos but have a different focus.  Tutorials are specifically built for beginners and are meant to help them gain experience with the product.  For example taking a course through the entire course lifecycle, even if there is no content, and it's not exactly how you would do it with a real course, gives a beginner a meaningful experience that helps them better navigate the product.
+**Quickstart**
```
Comment (sarina) [inline source/documentors/concepts/content_types.rst:6]: Consider: using subheadings so they can be linked to

File: source/documentors/concepts/content_types.rst [hunk 3]
```diff
@@ -3,13 +3,30 @@ Documentation Content Types
...
+**How-to** 
+
+How-to topics provide direct and easy-to-follow instructions to accomplish specific goals. For example, topics provide the steps to create a new course subsection and to schedule a course. 
+
+How-to document names should start with a verb to reflect action (e.g. ``create_new_course.rst``, ``schedule_course.rst``).
+
+
+**Reference** 
+
+Reference topics provide details about a function or feature of Open edX. For example there are many details about course subsections such as the different publication states, grading configuration, and visibility that are not included in the how-to topic *Create a Subsection* but are fully described in the reference topic *Course Subsections*.  These two topics are linked in See Also sections.
```
Comment (sarina) [inline source/documentors/concepts/content_types.rst:19]:
```suggestion
Reference topics provide details about a function or feature of the Open edX platform. For example there are many details about course subsections such as the different publication states, grading configuration, and visibility that are not included in the how-to topic *Create a Subsection* but are fully described in the reference topic *Course Subsections*.  These two topics are linked in See Also sections. 
```

File: source/documentors/concepts/content_types.rst [hunk 4]
```diff
@@ -3,13 +3,30 @@ Documentation Content Types
...
+**Reference** 
+
+Reference topics provide details about a function or feature of Open edX. For example there are many details about course subsections such as the different publication states, grading configuration, and visibility that are not included in the how-to topic *Create a Subsection* but are fully described in the reference topic *Course Subsections*.  These two topics are linked in See Also sections. 
+
+Reference document names should use nouns and clear descriptions of the subject matter, as these docs serve as lookup resources (e.g. ``publication_states.rst``, ``grading_configuration.rst``).
+
+
+**Concept** 
+
+Concept topics provide best practices or other guidelines for using Open edX. For example, the topics *Crafting Effective Learning Objectives* and *Aligning End-of-Module Assessments to Learning Objectives* are meant to provide guidelines for educators.
```
Comment (sarina) [inline source/documentors/concepts/content_types.rst:26]:
```suggestion
Concept topics provide best practices or other guidelines for using the Open edX platform. For example, the topics *Crafting Effective Learning Objectives* and *Aligning End-of-Module Assessments to Learning Objectives* are meant to provide guidelines for educators. 
```

File: source/documentors/references/doc_checklist.rst [hunk 5]
```diff
@@ -102,4 +120,52 @@ In the pull request, click **Details** to see the documentation with your change
 
 If there is an error when building the documentation, the pull request will indicate that checks failed, and the **Details** link will take you to the error message. If you are able to fix the issue indicated by the error message, do so; if not, contact the Open edX team for assistance.
 
+Migrating Docs Checklist
```
Comment (sarina) [inline source/documentors/references/doc_checklist.rst:123]: Should this be its own, temporary document?

File: source/links.txt [hunk 6]
```diff
@@ -21,6 +21,10 @@
 
 .. _How to start contributing to the Open edX code base: https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base
 
+.. _Submit a Docs Issue: https://github.com/openedx/docs.openedx.org/issues/new/choose
+
+.. _Open EdX Doc Migration Tracking: https://docs.google.com/spreadsheets/d/1r0mbC1Ay-6vGVzGvHKRq9VHyowT7j-lm-4-8z38c5mM/edit
```
Comment (sarina) [inline source/links.txt:26]:
```suggestion
.. _Open edX Doc Migration Tracking: https://docs.google.com/spreadsheets/d/1r0mbC1Ay-6vGVzGvHKRq9VHyowT7j-lm-4-8z38c5mM/edit
```

File: source/documentors/references/doc_checklist.rst [hunk 7]
```diff
@@ -102,4 +120,52 @@ In the pull request, click **Details** to see the documentation with your change
...
+************************
+
+Some checklist items are specific to migrating 2U/Edx.org legacy docs into Open edX® Docs. During migration, documentors are using this `Open EdX Doc Migration Tracking`_ sheet.
```
Comment (sarina) [inline source/documentors/references/doc_checklist.rst:126]:
```suggestion
Some checklist items are specific to migrating 2U/Edx.org legacy docs into Open edX® Docs. During migration, documentors are using this `Open edX Doc Migration Tracking`_ sheet.
```

File: source/documentors/references/doc_checklist.rst [hunk 8]
```diff
@@ -102,4 +120,52 @@ In the pull request, click **Details** to see the documentation with your change
...
+
+Remove or modify references that are specific to 2U/EdX.org
+========================================================================
+
+When migrating from legacy documenation from 2U/EdX.org, remove references that are applicable only to the 2U or EdX.org users.
```
Comment (pdpinch) [inline source/documentors/references/doc_checklist.rst:131]:
```suggestion
When migrating legacy documentation from 2U/EdX.org, remove references that are applicable only to the 2U or EdX.org users.
```

File: source/documentors/references/doc_checklist.rst [hunk 9]
```diff
@@ -102,4 +120,52 @@ In the pull request, click **Details** to see the documentation with your change
...
+
+Modify references that may have come from 2U/EdX.org but are also applicable to Open edX® LMS users.
```
Comment (pdpinch) [inline source/documentors/references/doc_checklist.rst:133]: I'm not sure why you mention LMS specifically here. If that's important, please ignore my comment.
```suggestion
Modify references that may have come from 2U/EdX.org but are also applicable to Open edX® users.
```

**Review by pdpinch — COMMENTED** [review]
Thanks for doing this John. I left a couple of minor copyedits.

File: source/documentors/references/doc_checklist.rst (hunk 9 above)
Comment (jswope00) [reply source/documentors/references/doc_checklist.rst:133]: I have been told, at times, to use Open edX as an adjective (so always have a noun like platform or LMS or community after it). So leaving this one as is.

**Pull request merged**